assets/src/application/ui/calendars/dateDisplay/TimezoneTimeInfo/index.tsx

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
-import classNames from 'classnames';
+import {
+	IconButton,
+	Popover,
+	PopoverTrigger,
+	PopoverContent,
+	PopoverHeader,
+	PopoverBody,
+	PopoverArrow,
+	PopoverCloseButton,
+} from '@chakra-ui/core';
 import { __ } from '@wordpress/i18n';
 import { GlobalOutlined } from '@ant-design/icons';
```

Comment:
I didn't found similar icon in chakra's lib, so left the andt until will find another